packages/backend/src/api/resources/CommercePlan.ts

```diff
@@ -1,16 +1,31 @@
 import { Feature } from './Feature';
 import type { CommercePlanJSON } from './JSON';
 
+/**
+ * The `CommerceMoneyAmount` type represents a monetary value with currency information.
+ * @type
+ */
 export type CommerceMoneyAmount = {
+  /**
+   * The raw amount as a number, usually in the smallest unit of the currency (like cents for USD). For example, `1000` for $10.00.
+   */
   amount: number;
+  /**
+   * The amount as a formatted string. For example, `10` for $10.00.
+   */
   amountFormatted: string;
+  /**
+   * The ISO currency code for this amount. For example, `USD` or `EUR`.
```

Comment:
Let's not add hints that we support multiple currencies.
```suggestion
   * The ISO currency code for this amount. For example, `USD`.
```